### PR TITLE
meta: Bug report template

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
meta: Bug report template

## :bulb: Motivation and Context
We'd like useful info in our bug reports, a bug template is helpful for that.

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] All tests passing
- [ ] No breaking changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -1,21 +1,39 @@
----
-name: Report a bug
-about: A bug you encountered with the sentry_flutter package.
----
+_Platform:_
```
Comment (marandaneto) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:7]:
```suggestion
---
name: Bug report
about: Create a report to help us improve

---

_Platform:_
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 2]
```diff
@@ -1,21 +1,45 @@
...
+- [ ] pub.dev
+- [ ] GitHub
 
+Output of the command `flutter doctor` below::
 
+The output goes here...
 
-2. Include the output of the command `flutter doctor` below:
+The version of the SDK (See pubspec.lock):
+**4.x.x**
```
Comment (bruno-garcia) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:31]: Should we add 4.x.x here? Maybe just leave it out?

Comment (marandaneto) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md:31]: we do with the other SDKs, no strong opinion, but it gives an idea to the user about the latest major version.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 3]
```diff
@@ -1,21 +1,45 @@
...
-1. Please describe the issue and how to reproduce it, or link to a small reproducible application:
+_split-debug-info and obfuscate (Flutter Android or iOS) or CanvasKit (Flutter Web):_
+- [ ]  Enabled
+- [ ]  Disabled
 
+_Platform installed with:_
+- [ ] pub.dev
+- [ ] GitHub
 
+Output of the command `flutter doctor` below::
```
Comment (ueman) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]: Shouldn't it be `flutter doctor -v`? Also double colons at the end.

Comment (marandaneto) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md]: good idea, I just copied over the old template, thanks @ueman :)

Comment (marandaneto) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]:
```suggestion
Output of the command `flutter doctor -v` below:
```

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**